dunamai/formatters.py

```diff
@@ -0,0 +1,153 @@
+from typing import Sequence, Union
+from dunamai import Version
+
+__all__ = ["pep440", "pep440_meta", "pep440_meta_id", "format_pieces_as_pep440"]
+
+
+def format_pieces_as_pep440(
```

Comment:
You could just use the existing `serialize_pep440`. The only difference is that `format_pieces_as_pep440` doesn't call `check_version`, but that shouldn't hurt anything here.